src/03-deals/deals.apib

```diff
@@ -22,11 +22,13 @@ Get a list of deals.
                 + id: `2659dc4d-444b-4ced-b51c-b87591f604d7` (string, required)
             + phase_id: `1758e847-cc1f-04ce-bd17-5a852e8c01b4` (string, optional)
             + estimated_closing_date: `2018-02-11` (string, optional, nullable)
+            + estimated_closing_date_before: `2020-01-31` (string, optional) - This date is inclusive
+            + estimated_closing_date_after: `2020-01-01` (string, optional) - This date is inclusive
             + One Of
                 + responsible_user_id: `98b2863e-7b01-4232-82f5-ede1f0b9db22` (string, optional)
                 + responsible_user_id: `98b2863e-7b01-4232-82f5-ede1f0b9db22`, `98b2863e-7b01-4232-82f5-ede1f0b9db24` (array[string], optional)
-            + updated_since: `2018-02-11T16:45:30+00:00` (string, optional)
-            + created_before: `2018-02-11T16:45:30+00:00` (string, optional)
+            + updated_since: `2018-02-11T16:45:30+00:00` (string, optional) - This date is inclusive
+            + created_before: `2018-02-11T16:45:30+00:00` (string, optional) - This date is inclusive
```

Comment:
Q: is this how the endpoint filters at the moment? I'm not sure if we should make or keep this inclusive. "After" or "before" does specify that that date and time is not included, right?

Reply:
It appears to be already implemented inclusively for these filters. This does behave differently than i.e. the filters for time tracking. 🤔